addon/helpers/build-secondary-nav-links.js

```diff
@@ -100,6 +100,14 @@ export default Ember.Helper.extend({  // Helper defined using a class, so can in
                 }
             );
         }
+        if(session.get('isAuthenticated')) {
+            links.HOME.unshift(
+                {
+                    name: 'eosf.navbar.myQuickFiles',
+                    href: serviceLinks.myQuickFiles
+                }
+            );
+        }
```

Comment:
Please make the above `if (!session.get('isAuthenticated'))` an else for this `if`